.codecov.yml

```diff
@@ -0,0 +1,20 @@
+# Team Yaml
+coverage:
+  round: down
+  precision: 5
+
+# Repository Yaml
+coverage:
+  round: up
+  range: 0..10
+
+# Used in Codecov after updating
+coverage:
```

Comment:
It seems that this block overrides all the previous settings so I think we can remove them. At least that's how I interpret the validation output:
```shell
[archibald ⚡ /tmp] » curl --data-binary @codecov.yml https://codecov.io/validate
Valid!

{
  "ignore": [
    "^examples/data.*", 
    "^examples/imgs.*"
  ], 
  "coverage": {
    "range": [
      0.0, 
      10.0
    ], 
    "round": "up", 
    "precision": 5
  }
}
```
Also, the `range` parameter seems to define the region between which coverage status color transitions from red to green, see https://docs.codecov.io/docs/coverage-configuration#section-range. So `range: 0..10` is probably a little premature :joy: Maybe we can change the range to `50..100` or `50..90`. I don't have a strong opinion regarding the precision.